Nessus 09062017

erpadmin opened the pull request:
recent commits some of which attempt to address a few of the open issues

Commit:
implement /var/tmp handling, runs if rhel7cis_vartmp_src is defined

Commit:
fixed check for rhnsd daemon status

Commit:
5.6 regexp was checking for commented line

Commit:
4.2.1.3 escape $

Commit:
added and made use of clientaliveinterval variable

Commit:
improve /var/tmp handling

Commit:
unset vartmp variables

shepdelacreme reviewed (changes requested):
The idempotency check is failing because of some of the outstanding issues related to non-idempotent tasks. So you can ignore that failure for now. See my other comments for some potential improvements.

Commit:
recommended changes

Commit:
fix idempotency for rules 4.1.1.1/4.1.1.3

Commit:
add daswars workaround on rule 5.1.8

2 items were not idempotent, this is a known issue for file touch
https://github.com/ansible/ansible-modules-core/issues/170
